frontend/constants/networks.ts

```diff
@@ -25,6 +27,10 @@ const NETWORKS = {
     rpcUrl: 'https://rpc.testnet.lukso.network',
     explorerURL: 'https://explorer.execution.testnet.lukso.network',
     universalGraveForwarder: '0x7e77c704be7a6bba042f66eaba0b9557a872902d',
+    previousGraveForwarders: [
+      '0x1e2f66d2dC19bB13617D1d06CBb3E8d8aa4567e3',
```

Comment:
you may have to categorize them to find the right ABI to use
(ex: v1: 0x43wedfasd, V2: 0x0430239)

Reply:
As long as the getGrave() function remains the same we should be okay